db/re/skill_db.yml

```diff
@@ -19243,6 +19245,7 @@ Body:
         Time: 16000
       - Level: 5
         Time: 18000
+    Duration2: 5000
```

Comment:
I'm a bit confused by this commit. Isn't Stone Curse the only skill with a 5000ms incubation time? Last time I checked the incubation time for all other skills was ~100ms. See #990.

Reply:
Correct. I moved the 5 second incubation time to the skill_db since the status is separated now.